### PR TITLE
corpora.json for future development added, Magda added to About Us

### PR DESCRIPTION
**Pull request by magdawalkiewicz25**

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -131,6 +133,21 @@ <h3 class="about-name-surname">Krystian Borkowski</h3>
                             ></a>
                         </div>
                     </div>
+                    <div class="col-12 col-md-4">
```
Comment (brkutd) [inline html/about.html:136]: i'd think about rearranging the columns as col-12 is the maximum and with 4x col-md-4 we've got sth like col-16, which won't work well probably, so we can divide everything into 2 big columns col-md-6 (probably), or into 4 smaller ones (col-md-3)

**Review by klaudiaceglarz — DISMISSED** [review]
good job, Magda!

File: html/about.html (hunk 1 above)
Comment (magdawalkiewicz25) [reply html/about.html:136]: good idea, I"ll change it right away!

File: js/corpora.json [hunk 2]
```diff
@@ -1,8 +1,80 @@
 {
   "main": {
-    "accept": ["yes", "yup"],
-    "decline": ["no", "nope"],
-    "dialogue_start": ["i feel really anxious today", "i have so much work", "i can not deal with the pressure and stress"],
-    "calming_scenarios": [[], [], []]
+    "accept": ["yes", "yup", "yeah", "sure", "definitely", "right", "that is right", "alright", "fine", "mhm"],
+    "decline": ["no", "nope", "i do not know", "i have not"],
```
[the remark this answers is not in the document]
Comment (magdawalkiewicz25) [reply js/corpora.json]: new options are added :)

**Review by brkutd — APPROVED** [review]

**Pull request merged**